### PR TITLE
Add support for Fast calls with Uint8Array

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by ry — APPROVED** [review]
LGTM

File: src/fast_api.rs [hunk 1]
```diff
@@ -78,6 +78,7 @@ pub enum SequenceType {
 pub enum CType {
   Void = 0,
   Bool,
+  Uint8,
```
Comment (ry) [inline src/fast_api.rs:81]: This has the correct value ?

Comment (bartlomieju) [reply src/fast_api.rs:81]: Yes, tests are passing locally

File: tests/test_api.rs [hunk 2]
```diff
@@ -7227,6 +7227,80 @@ fn test_fast_calls_arraybuffer() {
...
+#[test]
+fn test_fast_calls_typedarray() {
+  static mut WHO: &str = "none";
+  fn fast_fn(
+    _recv: v8::Local<v8::Object>,
+    data: *const fast_api::FastApiTypedArray<u8>,
+  ) -> u32 {
+    unsafe { WHO = "fast" };
+    let sum = unsafe { &*data }.get(0) + unsafe { &*data }.get(1) +
+      unsafe { &*data }.get(2);
```
Comment (ry) [inline tests/test_api.rs:7239]: Check values here with assert_eq

**Pull request merged**